File: app/src/main/java/com/slutsenko/cocktailapp/network/CocktailList.java
package com.slutsenko.cocktailapp.network;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
import com.slutsenko.cocktailapp.Cocktail;

import java.util.ArrayList;

public class CocktailList {

    @SerializedName("drinks")
    @Expose
    private ArrayList<Cocktail> drinks = new ArrayList<Cocktail>();

    public ArrayList<Cocktail> getCocktails() {
        return drinks;
    }
}

File: app/src/main/java/com/slutsenko/cocktailapp/network/CocktailList.kt
package com.slutsenko.cocktailapp.network

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import com.slutsenko.cocktailapp.entity.Cocktail
import java.util.*

class CocktailList {
    @SerializedName("drinks")
    @Expose
    val cocktails = ArrayList<Cocktail>()
}